### PR TITLE
Move AMD's MP0_C2P_MSG_37 to registers folder and add it to marshaller

### PR DESCRIPTION
**Pull request by rihter007**
Fix PSP->PSB, PSB stands for Platform secure boot

### CODE REVIEW

File: pkg/pcr/flow.go [hunk 1]
```diff
@@ -42,10 +42,10 @@ func FlowFromString(s string) (Flow, error) {
 		return FlowIntelLegacyTXTEnabledTPM12, nil
 	case "cbnt0t":
 		return FlowIntelCBnT0T, nil
-	case "legacypspdisabled":
-		return FlowLegacyPSPDisabled, nil
-	case "legacypspenabled":
-		return FlowLegacyPSPEnabled, nil
+	case "legacypsbdisabled":
+		return FlowLegacyPSBDisabled, nil
+	case "legacypsnenabled":
```
Comment (xaionaro) [inline pkg/pcr/flow.go:47]: Looks like a typo (see `psn`).

**Review by xaionaro — CHANGES_REQUESTED** [review]
Please fix a typo

File: pkg/pcr/flow.go (hunk 1 above)
Comment (rihter007) [reply pkg/pcr/flow.go:47]: Argh, thanks!

**Comment by xaionaro** [comment]
Not approving, because fixing the typo will require re-approve. I'll try to approve as soon as the typo is fixed.

**Review by xaionaro — APPROVED** [review]

**Review by ChriMarMe — APPROVED** [review]

**Pull request merged**